templates/repo/issue/view_content/context_menu.tmpl

```diff
@@ -0,0 +1,22 @@
+{{if .ctx.IsSigned}}
+<div class="item action ui pointing top right custom dropdown context-dropdown">
+	<a class="context-menu">
+		<i class="fa fa-ellipsis-h"></i>
+	</a>
+	<div class="menu">
+		{{$url := (Printf "%s%s/issues/%d#%s" AppUrl .ctx.Repository.FullName .ctx.Issue.Index .item.HashTag)}}
+		{{if .diff}}
```

Comment:
Perhaps it's better to go for an if/else/end construct and avoid executing template code unless it's required.

Reply:
imo it looks cleaner, but I don't know whether it has an effect on rendering.  
I've changed it to use an if-else structure.